ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/InstallPlaywrightBrowsersTask.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+using YamlDotNet.Serialization;
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks
+{
+    public class InstallPlaywrightBrowsersTask : GithubTask
+    {
+        [YamlIgnore]
+        public string ProjectName { get; set; }
+        public override string Run => $"pwsh ./{ProjectName}/bin/Debug/net9.0/playwright.ps1 install";
```

Comment:
Add documentation to show the default command so consumers can know when to override it with additional arguments

Reply:
Just using ProjectName as a property is not enforcing its use.  I would suggest taking that value in via a constructor so consumer does not have to figure out what is needed.

```cs
    public class InstallPlaywrightBrowsersTask : GithubTask
    {
        public InstallPlaywrightBrowsersTask(string projectName)
        {
            this.projectName = projectName;
        }
        
        public override string Run => $"pwsh ./{projectName}/bin/Debug/net9.0/playwright.ps1 install";
    }
```

Reply:
I also added .NET version as parameter:

```cs
 public InstallPlaywrightBrowsersTask(
     string projectName,
     string dotnetVersion = "net9.0")
 {
     this.projectName = projectName;
     this.dotnetVersion = dotnetVersion;
 }
```